cpp/ps0000/ps0000_soin_cpp.cpp

```diff
@@ -1,8 +1,9 @@
-#include<iostream>
-#include<conio.h>
+#include <iostream> // 2/2
+#include <conio.h>
 using namespace std;
-int main()
+int main() // 1/1
 {
-	cout << "hello ,world! ";
-	_getch();
-}
\ No newline at end of file
+	cout << "Hello, World!\n"; // 3/4
```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [ ] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line